### PR TITLE
Added time window & length window benchmark tests & removed histogram

### PR DESCRIPTION
**Pull request by BraveenSri**

### CODE REVIEW

File: EsperBenchmark/FilterBenchmark/pom.xml [hunk 1]
```diff
@@ -5,12 +5,25 @@
     <parent>
         <artifactId>Benchmark</artifactId>
         <groupId>Esper</groupId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0</version>
```
Comment (miyurud) [inline EsperBenchmark/FilterBenchmark/pom.xml]: We should keep the snapshot label

File: EsperBenchmark/TimeWindowBenchmark/src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import com.espertech.esper.common.client.EPCompiled;
```
Comment (miyurud) [inline EsperBenchmark/TimeWindowBenchmark/src/main/java/Main.java]: Lets have a meaningful class name than Main.java

File: SiddhiBenchmark/FilterBenchmark/src/main/java/Main.java [hunk 3]
```diff
@@ -1,15 +1,10 @@
-import java.io.BufferedReader;
```
Comment (miyurud) [inline SiddhiBenchmark/FilterBenchmark/src/main/java/Main.java:1]: Lets have a meaningful name than Main.java

File: SiddhiBenchmark/LengthWindowBenchmark/src/main/java/Main.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
+import java.io.File;
```
Comment (miyurud) [inline SiddhiBenchmark/LengthWindowBenchmark/src/main/java/Main.java]: Lets have meaningful file name than Main.Java

File: EsperBenchmark/FilterBenchmark/src/main/java/DataGenerator.java [hunk 5]
```diff
@@ -1,4 +1,5 @@
 public class DataGenerator {
```
Comment (miyurud) [inline EsperBenchmark/FilterBenchmark/src/main/java/DataGenerator.java:14]: Lets add the copyright header for all the java source files.

**Review by miyurud — CHANGES_REQUESTED** [review]

File: EsperBenchmark/FilterBenchmark/pom.xml (hunk 1 above)
Comment (BraveenSri) [reply EsperBenchmark/FilterBenchmark/pom.xml]: Reversed the version to 1.0-SNAPSHOT

File: EsperBenchmark/TimeWindowBenchmark/src/main/java/Main.java (hunk 2 above)
Comment (BraveenSri) [reply EsperBenchmark/TimeWindowBenchmark/src/main/java/Main.java]: Changed the class name to Benchmark

File: SiddhiBenchmark/FilterBenchmark/src/main/java/Main.java (hunk 3 above)
Comment (BraveenSri) [reply SiddhiBenchmark/FilterBenchmark/src/main/java/Main.java:1]: Changed the class name to Benchmark

File: SiddhiBenchmark/LengthWindowBenchmark/src/main/java/Main.java (hunk 4 above)
Comment (BraveenSri) [reply SiddhiBenchmark/LengthWindowBenchmark/src/main/java/Main.java]: Changed the class name to Benchmark

File: EsperBenchmark/FilterBenchmark/src/main/java/DataGenerator.java (hunk 5 above)
Comment (BraveenSri) [reply EsperBenchmark/FilterBenchmark/src/main/java/DataGenerator.java:14]: Added copyright header

**Pull request closed without merging**